### ISSUE TITLE
Newly [2017-06-15] not running on ArchLinux

### ISSUE THREAD

**Issue by molaeiali** [ilius/starcal#40]
Installed from AUR, after running from terminal, I get:
```
** (starcal.py:4653): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-ltHD5mW8Vn: Connection refused
Traceback (most recent call last):
  File "/usr/share/starcal3/scal3/ui_gtk/starcal.py", line 1075, in <module>
    sys.exit(main())
  File "/usr/share/starcal3/scal3/ui_gtk/starcal.py", line 1053, in main
    ui.init()
  File "/usr/share/starcal3/scal3/ui.py", line 531, in init
    eventGroups = event_lib.EventGroupsHolder.load()
  File "/usr/share/starcal3/scal3/s_object.py", line 132, in load
    obj.setData(data)
  File "/usr/share/starcal3/scal3/event_lib.py", line 3851, in setData
    obj.save()
  File "/usr/share/starcal3/scal3/event_lib.py", line 2673, in save
    EventContainer.save(self)
  File "/usr/share/starcal3/scal3/event_lib.py", line 91, in save
    return BsonHistObj.save(self, *args)
  File "/usr/share/starcal3/scal3/s_object.py", line 264, in save
    history = self.loadHistory()
  File "/usr/share/starcal3/scal3/s_object.py", line 235, in loadHistory
    lastBasicData = self.loadBasicData()
  File "/usr/share/starcal3/scal3/s_object.py", line 233, in loadBasicData
    return jsonToData(open(self.file).read())
  File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

```

aur/starcal                                            3.0.6-3
extra/python                                         3.6.1-1
extra/python-gobject                            3.24.1-1
extra/python-cairo                                1.13.3-1
community/python-dateutil                   2.6.0-2
community/python-httplib2                   0.10.3-1
community/python-psutil                      5.2.2-1
community/python-pymongo                3.4.0-2

Above are versions of package and dependencies...

**Comment by ilius** [comment]
AUR package is not maintained by me.
Please use the `install-archlinux` script (as mentioned in README) from the latest `master` branch
